sdk/subnetid.go

```diff
@@ -1,165 +1,211 @@
 package sdk
 
 import (
+	"bytes"
+	"hash/fnv"
 	"path"
-	"path/filepath"
+	"strconv"
 	"strings"
 
 	"github.com/filecoin-project/go-address"
 )
 
 type SubnetID struct {
-	Parent string
-	Actor  address.Address
+	// ChainID of the root network
+	Root uint64
+	// Children up to the current subnet.
+	Children []address.Address
 }
 
-var id0, _ = address.NewIDAddress(0)
-
 const (
-	RootStr          = "/root"
+	RootPrefix       = "r"
 	SubnetSeparator  = "/"
 	UndefStr         = ""
```

Comment:
This const is not used.

Reply:
Yes it is, as part of the `String()` implementation